#  Machine Learning (ML) for Spatially Resolved Transcriptomics (SRT) with Histology Images

## Papers (2021-)



| Date     | First Author                                                 | Title                                                        | Journal                                                      | Code                                                 |
| -------- | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ | ---------------------------------------------------- |
| 20220321 |  [Runmin Wei](https://scholar.google.com.hk/citations?user=chbmcD0AAAAJ&hl=en&oi=sra)               | Spatial charting of single-cell transcriptomes in tissues | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01233-1)  | [R](https://github.com/navinlabcode/CellTrek) |
| 20220131 | [Giovanni Palla](https://scholar.google.com.hk/citations?user=20uwxzkAAAAJ&hl=en&oi=sra) | Squidpy: a scalable framework for spatial single cell analysis | [Nature Methods](https://www.nature.com/articles/s41592-021-01358-2) | [Python](https://github.com/theislab/squidpy)        |
| 20220113 | [Vitalii Kleshchevnikov](https://github.com/vitkl/)     | Cell2location maps fine-grained cell types in spatial transcriptomics | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01139-4) | [PyTorch](https://github.com/BayraktarLab/cell2location)         |
| 20211207 | [Stephan Preibisch](https://preibischlab.mdc-berlin.de/)     | Image-based representation of massive spatial transcriptomics datasets | [bioRxiv](https://www.biorxiv.org/content/10.1101/2021.12.07.471629v1) | [JAVA](https://github.com/PreibischLab/STIM)         |
| 20211129 | Ludvig Bergenstr√•hle                                         | Super-resolved spatial transcriptomics by deep data fusion   | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01075-3) | [PyTorch](https://github.com/ludvb/xfuse)            |
| 20211125 | [Maria Brbic](https://cs.stanford.edu/~mbrbic/)              | Annotation of Spatially Resolved Single-cell Data with STELLAR | [bioRxiv](https://www.biorxiv.org/content/10.1101/2021.11.24.469947v1.full) |                                                      |
| 20211110 | [Brendan F. Miller](https://scholar.google.com/citations?hl=en&user=BP7VBw0AAAAJ&view_op=list_works&sortby=pubdate) | Reference-free cell-type deconvolution of multi-cellular pixel-resolution spatially resolved transcriptomics data | [bioRxiv](https://www.biorxiv.org/content/10.1101/2021.06.15.448381v2) | [R](https://github.com/JEFworks-Lab/STdeconvolve)    |
| 20211108 | Tamas Ryszard Sztanka-Toth                                   | Spacemake: processing and analysis of large-scale spatial transcriptomics data | [bioRxiv](https://www.biorxiv.org/content/10.1101/2021.11.07.467598v1) | [Python](https://github.com/rajewsky-lab/spacemake)  |
| 20211028 | [Tommaso Biancalani](https://scholar.google.com/citations?user=s_qd9x0AAAAJ&hl=en&oi=sra) | Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram | [Nature Methods](https://www.nature.com/articles/s41592-021-01264-7) | [Pytorch](https://github.com/broadinstitute/Tangram) |
| 20211028 | [Jian Hu](https://scholar.google.com/citations?user=aFlfjvwAAAAJ&hl=en&oi=sra) | SpaGCN: Integrating gene expression, spatial location and histology to identify spatial domains and spatially variable genes by graph convolutional network | [Nature Methods](https://www.nature.com/articles/s41592-021-01255-8) | [PyTorch](https://github.com/jianhuupenn/SpaGCN)     |
| 20211014 | [Viktor Petukhov](https://scholar.google.com/citations?user=rkEysdsAAAAJ&hl=en&oi=sra) | Cell segmentation in imaging-based spatial transcriptomics   | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01044-w) | [Julia](https://github.com/kharchenkolab/Baysor)     |
| 20210603 | [Edward Zhao](https://scholar.google.com.hk/citations?user=ayOCjW8AAAAJ&hl=en&oi=sra) | Spatial transcriptomics at subspot resolution with BayesSpace | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-00935-2) | [R](https://github.com/edward130603/BayesSpace)      |




## Talks
- 20220111-[Fabian J. Theis](https://www.helmholtz-munich.de/icb/institute/staff/staff/ma/2494/index.html): Learning cellular state and dynamics in single cell genomics ([YouTube](https://www.youtube.com/watch?v=_3s8g7t6Nww))
- Spatial Omics Zoom Seminar Series ([Homepage](https://spatialomics.net/), [YouTube](https://www.youtube.com/channel/UCVCbNOLVh325dieVdnF-aWg))
- 20211011-[Joakim Lundeberg](https://www.kth.se/profile/joalun): Spatial Transcriptomics and its application in skin research ([YouTube](https://www.youtube.com/watch?v=WjwvJtm1F3A))

- 20210810-[Edward Zhao](https://scholar.google.com.hk/citations?user=ayOCjW8AAAAJ&hl=en&oi=sra): BayesSpace enables the robust characterization of spatial gene expression architecture ([YouTube](https://www.youtube.com/watch?v=vQRT8myi-qs))
- 20210607-[Omer Bayraktar](https://www.sanger.ac.uk/person/bayraktar-omer/): Mapping neural cell type diversity using spatial transcriptomics ([YouTube](https://www.youtube.com/watch?v=qtADS6IHQGE))

- 20210327-[Fei Chen](https://www.insitubiology.org/research): Next generation tools for spatial genomics ([YouTube](https://www.youtube.com/watch?v=HmYuS1qHuew&t=2983s))

- 20210105-[Xiaowei Zhuang](http://zhuang.harvard.edu/): Single-Cell Transcriptome Imaging and Brain Cell Atlas ([YouTube](https://www.youtube.com/watch?v=a9_rGdlM_Rw&t=2165s))

- 20201123-[James Zou](https://www.james-zou.com/): Deep learning to integrate histology with spatial transcriptomics ([YouTube](https://www.youtube.com/watch?v=9PYeMn8t57Y))


## Public Datasets

[GoogleDocs](https://docs.google.com/spreadsheets/d/1LRw5D-NAFhIlYbdP7v3i_IICBLFDs0vvha56OKnCPwg/edit?usp=sharing)



## Reviews/Perspectives/Editorials



| Date     | First Author                                                 | Title                                                        | Journal                                                      |
| -------- | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
| 20220207 | [Giovanni Palla](https://scholar.google.com.hk/citations?user=20uwxzkAAAAJ&hl=en&oi=ao) | Spatial components of molecular tissue biology | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01182-1) |
| 20210811 | [Anjali Rao](https://scholar.google.ca/citations?user=_QwXNM8AAAAJ&hl=en&oi=sra) | Exploring tissue architecture using spatial transcriptomics  | [Nature](https://www.nature.com/articles/s41586-021-03634-9) |
| 20211018 | [Kevin M. Boehm](https://scholar.google.ca/citations?hl=en&user=3t7ftQwAAAAJ&view_op=list_works&alert_preview_top_rm=2&sortby=pubdate) | Harnessing multimodal data integration to advance precision oncology | [Nature Reviews Cancer](https://www.nature.com/articles/s41568-021-00408-3) |
| 20210601 | [Jian Hu](https://scholar.google.com/citations?user=aFlfjvwAAAAJ&hl=en&oi=sra) | Statistical and machine learning methods for spatially resolved transcriptomics with histology | [Computational and Structural Biotechnology Journal](https://www.sciencedirect.com/science/article/pii/S2001037021002907?via%3Dihub) |
| 20210106 | -                                                            | Method of the Year 2020: spatially resolved transcriptomics  | [Nature Methods](https://www.nature.com/articles/s41592-020-01042-x) |

